sh/tests/append-output_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Your solution doesn't pass the tests. When i run this test file with your solution and the results.txt given on the usage section I get the following:
![image](https://user-images.githubusercontent.com/18433970/212690765-8bba8163-0e8d-4d22-8eca-7ebf0fddf6fb.png)

Reply:
fixed